src/code/views/value-slider-view.tsx

```diff
@@ -1,8 +1,18 @@
 import * as React from "react";
 const log = require("loglevel");
 
+import { urlParams } from "../utils/url-params";
 import { tr } from "../utils/translate";
 
+const urlValueWithDefault = (name: string, defaultValue: number, urlValue?: string) => {
```

Comment:
Seems like this function `urlValueWithDefault` should exist in `urlParams` util package?

Reply:
I shouldn't have asked for a review for this PR yet - this function was in to allow the building models folks to tweak the final internal settings without me rebuilding it - this will all be removed.